examples/plot_barber2020_simulations.py

```diff
@@ -103,61 +123,75 @@ def plot_simulation_results(results: Dict, methods: List[str], title: str) -> No
         Title of the plot.
     """
     fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(15, 5))
-    plt.rcParams.update({'font.size': 14})
+    plt.rcParams.update({"font.size": 14})
     if title is not None:
         plt.suptitle(title)
     for method in methods:
-        coverage_mean = []
-        coverage_SE = []
-        for dim_val in dim_vals:
+        coverage_mean, coverage_SE = [], []
+        width_mean, width_SE = [], []
+        for dimension in dimensions:
             coverage_mean.append(np.array([
-                results[key][3] for key in results
-                if (results[key][2] == method) & (results[key][1] == dim_val)
+                results[key]["coverage"] for key in results
+                if (
+                    (results[key]["method"] == method) &
+                    (results[key]["dimension"] == dimension)
+                )
             ]).mean())
             coverage_SE.append(np.array([
-                results[key][3] for key in results
-                if (results[key][2] == method) & (results[key][1] == dim_val)
-            ]).std()/np.sqrt(ntrial))
-        coverage_mean = np.array(coverage_mean)
-        coverage_SE = np.array(coverage_SE)
-        ax1.plot(dim_vals, coverage_mean, label=method)
-        ax1.fill_between(
-            dim_vals, coverage_mean-coverage_SE, coverage_mean+coverage_SE, alpha=0.25
-        )
-    ax1.axhline(1-alpha, linestyle='dashed', c='k')
-    ax1.set_ylim(0.0, 1.0)
-    ax1.set_xlabel('Dimension d')
-    ax1.set_ylabel('Coverage')
-    ax1.legend()
-    for method in methods:
-        width_mean = []
-        width_SE = []
-        for dim_val in dim_vals:
+                results[key]["coverage"] for key in results
+                if (
+                    (results[key]["method"] == method) &
+                    (results[key]["dimension"] == dimension)
+                )
+            ]).std(ddof=np.sqrt(ntrial)))
```

Comment:
Attention à bien lire la documentation de `ddof` :  

ddofint, optional
Means Delta Degrees of Freedom. The divisor used in calculations is N - ddof, where N represents the number of elements. By default ddof is zero.

By the way, je ne comprends par pourquoi il faut changer le diviseur dans `std` ? A ajouter comme point oral à notre daily

Reply:
Je dois t'avouer que ce point n'est pas très clair pour moi non plus, j'avais repris le notebook du papier qui normalisait par `sqrt(n_trials)`. Discutons-en demain.

Reply:
J'ai à nouveau normaliser par `sqrt(n_trials)` afin d'expliciter la normalisation pour le calcul de l'erreur type.